src/NoteWorkspace.ts

```diff
@@ -342,13 +349,17 @@ export class NoteWorkspace {
     return t.match(this.rxFileExtensions()) ? t : `${t}.${this.defaultFileExtension()}`;
   }
 
-  static newNote(context: vscode.ExtensionContext) {
-    // console.debug('newNote');
-    const inputBoxPromise = vscode.window.showInputBox({
+  static showNewNoteInputBox() {
```

Comment:
This input box is common to `newNote` and `newNoteFromSelection`, so I broke it out into its own method.